rdrf/rdrf/db/filestorage.py

```diff
@@ -1,3 +1,5 @@
+import uuid
```

Comment:
The `Content-disposition` header uses the new uuid as the file name when downloading the file. I think we can modify `get_file` to use this.

Reply:
Fixed. Good pick up :+1: